packages/where/index.js

```diff
@@ -0,0 +1,88 @@
+// @ts-check
+
+/* Infers the rendezvous path for the endo.sock file and apps from the platform
+ * and environment.
+ */
+
+/**
+ * @param {{[name: string]: string}} env
+ */
+const whereEndoWindows = env => {
+  // Favoring local app data over roaming app data since I don't expect to be
+  // able to listen on one host and connect on another.
+  if (env.LOCALAPPDATA !== undefined) {
+    return `${env.LOCALAPPDATA}\\Endo`;
+  }
+  if (env.APPDATA !== undefined) {
+    return `${env.APPDATA}\\Endo`;
+  }
+  if (env.USERPROFILE !== undefined) {
+    return `${env.USERPROFILE}\\AppData\\Endo`;
```

Comment:
If you like the `String.raw` suggestion, you could apply it here and below.
```suggestion
    return String.raw`${env.USERPROFILE}\AppData\Endo`;
```